dask/dataframe/tests/test_shuffle.py

```diff
@@ -1538,3 +1538,14 @@ def test_sort_values_timestamp(npartitions):
     result = ddf.sort_values("time")
     expected = df.sort_values("time")
     assert_eq(result, expected)
+
+
+def test_sort_values_with_null_partition():
+    df = pd.DataFrame({"a": [2, 3, 1, 2, None, None]})
+    df["a"] = df["a"].astype("Int64")
```

Comment:
Thanks @charlesbluca! What about other extension dtypes? Specifically, I'm wondering about `Int64`, `Float64`, `int64[pyarrow]`, and `float64[pyarrow]`